app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] good job adding this line, however, you can remove the foreign_key part this is only set in the has_many part of the relationship.